shell/platform/windows/flutter_windows_texture_registrar_unittests.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <iostream>
+
+#include "flutter/shell/platform/windows/flutter_windows_engine.h"
+#include "flutter/shell/platform/windows/flutter_windows_texture_registrar.h"
+#include "gtest/gtest.h"
+
+namespace flutter {
+namespace testing {
+
+TEST(FlutterWindowsTextureRegistrarTest, CreateDestroy) {
+  FlutterDesktopEngineProperties props = {};
+  props.assets_path = L"FakeAssets";
+  props.icu_data_path = L"icudtl.dat";
+
+  FlutterProjectBundle bundle(props);
+  FlutterWindowsEngine engine(bundle);
+
+  FlutterWindowsTextureRegistrar registrar(&engine);
+
+  EXPECT_TRUE(true);
+}
+
+// TODO Mock FlutterEngineRegisterExternalTexture etc.
+// TEST(FlutterWindowsTextureRegistrarTest, RegisterTexture) {
+//   EXPECT_TRUE(true);
+// }
```

Comment:
We don't generally land commented out code. It's fine to include a TODO for more tests explaining what they should cover and what's needed to implement them, but the commented out stub of a test function isn't really useful.